CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - CPU usage spikes due to mouse movements for unfocused windows on X11/Windows
 - First window on macOS not tabbed with system prefer tabs setting
 - Window being treaten as focused by default on Wayland
+- `--help` output for `--class` does not match man pages
```

Comment:
Please add this to the 0.12.0 changelog, since 0.11.0 was already released.

Reply:
That's embarrassing, thanks for being patient with me, it should be fixed now.